Orange/widgets/unsupervised/owtsne.py

```diff
@@ -258,6 +265,9 @@ def pca_preprocessing(self):
     def __start(self):
         self.pca_preprocessing()
 
+        # Dirty flag indicating whether the embedding has been drawn at least once
+        self.needs_to_draw = True
```

Comment:
This can probably be removed now. In any case the comment is no longer true.

Reply:
This cannot be removed because, at the start of the optimization, we have to draw *something*, so we just draw some random points, because the graph widget *needs* some data to be shown. However, we'd like to replace this random data with something slightly more meaningful as soon as possible, so setting this to `True` here will ensure that after the first batch of iterations is done, the widget will display something better than random data.